scripts/contentScript.js

```diff
@@ -5,6 +5,48 @@ const keyToLabel = Object.freeze({
 	'important': "Important"
 })
 
+async function cssForGithubTrackRepo(trackedRepos) {
+	const allOrgRepo = document.getElementById('org-repositories');
+	const ignoreUrl = [
+		'stargazers',
+		'forks',
+		'issues',
+		'pulls',
+		'issues?q=label%3A%22help+wanted%22+is%3Aissue+is%3Aopen',
+		'repositories?type=all',
+		'issues?q=label%3A%22Help+wanted%22+is%3Aissue+is%3Aopen'
```

Comment:
These three look very sensitive - it seems that they might not cover all the cases

Reply:
I alreayd checked all these are same in every organization repo.